pages/t/[link].tsx

```diff
@@ -0,0 +1,39 @@
+import { GetServerSideProps } from 'next';
+import { useRouter } from 'next/router';
+import React from 'react';
+import Layout from '../../components/Layout';
+import styles from '../../styles/Page.module.scss';
+import {linksQuery, fetchContentful} from '../../utils';
+
+export default function LinkPage(): JSX.Element {
+  const router = useRouter();
+
+  if (typeof window !== 'undefined' && router.isFallback) {
+    void router.push('/');
+  }
+
+  return (
+    <Layout
+      content={(
+        <div id={styles.container}>
+          <a href="/">go back to home</a>
+        </div>
+      )}
+    />
+  );
+}
+
+export const getServerSideProps: GetServerSideProps = async ({ params, res}) => {
+  const {link} = params;
+  const contentfulRes = await fetchContentful(linksQuery(link));
+  const {data} = await contentfulRes.json();
+  const linkData = data?.linkCollection?.items;
+
+  if(linkData.length)
+  {
```

Comment:
nit: just for consistency

> nit means u can either accept or deny my suggestion (I wont be hurt if u say no)

```suggestion
  if(linkData.length) {
```